clab/file.go

```diff
@@ -27,9 +27,9 @@ const (
 	varFileSuffix = "_vars"
 )
 
-// GetTopology parses the topology file into c.Conf structure
+// LoadTopology parses the topology file into c.Conf structure
 // as well as populates the TopoFile structure with the topology file related information.
-func (c *CLab) GetTopology(topo, varsFile string) error {
+func (c *CLab) LoadTopology(topo, varsFile string) error {
```

Comment:
let's name it `LoadTopologyFromFile` to indicate that we work on the file input here

Reply:
Done